src/modules/wrapper/wrapper.component.spec.ts

```diff
@@ -0,0 +1,182 @@
+import { Observable } from 'rxjs/Observable';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ActivatedRoute, Router } from '@angular/router';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+
+import { expect } from '@blackbaud/skyux-builder/runtime/testing/browser';
+import { SkyAppWindowRef } from '@blackbaud/skyux-builder/runtime';
+
+import { StacheWrapperComponent } from './wrapper.component';
+import { StacheLayoutComponent } from '../layout/layout.component';
+import { StacheWrapperTestComponent } from './fixtures/wrapper.component.fixture';
+import { StacheTitleService } from './title.service';
+
+describe('Stache Wrapper Component', () => {
```

Comment:
Our other specs don't include spaces between the words `StacheWrapperComponent`.